### PR TITLE
[PowerMock]LPS 151648 

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @tinatian @dantewang,

This pr is to remove PowerMock (**Misc**) part one. Thanks.

https://issues.liferay.com/browse/LPS-151648

`modules/apps/..dynamic-data-mapping-expression`
`modules/apps/..asset-auto-tagger-service`
`modules/apps/..data-engine-api`
`modules/apps/..item-selector-web`
`modules/apps/..message-boards-service`
`modules/apps/..message-boards`
`modules/apps/..calendar-service`
`modules/apps/..google-places`

### CODE REVIEW

File: modules/apps/data-engine/data-engine-api/src/test/java/com/liferay/data/engine/field/type/util/LocalizedValueUtilTest.java [hunk 1]
```diff
@@ -30,31 +30,27 @@
...
-
-import org.powermock.api.mockito.PowerMockito;
+import org.mockito.Mockito;
 
 import org.skyscreamer.jsonassert.JSONAssert;
 
 /**
  * @author Mateus Santana
  */
-@RunWith(MockitoJUnitRunner.class)
```
Comment (dantewang) [inline modules/apps/data-engine/data-engine-api/src/test/java/com/liferay/data/engine/field/type/util/LocalizedValueUtilTest.java:48]: @JanisZhang What's the reason to remove `@RunWith(MockitoJUnitRunner.class)` here, but keep it in `DDMExpressionFactoryImplTest`?

**Review by dantewang — APPROVED** [review]
@JanisZhang Please see my comment and make changes if needed, thanks!

File: modules/apps/data-engine/data-engine-api/src/test/java/com/liferay/data/engine/field/type/util/LocalizedValueUtilTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/data-engine/data-engine-api/src/test/java/com/liferay/data/engine/field/type/util/LocalizedValueUtilTest.java:48]: @dantewang thanks... I forgot.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Pull request closed without merging**